src/index.html

```diff
@@ -11,6 +11,17 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#site" class="form form--01"
+     method="post">
+      <input type="text" class="form__textbox form--01__textbox"
```

Comment:
```suggestion
      <input type="text" class="form__textbox form--small__textbox"
```
this modifier is more descriptive 😄